src/components/layoutButton/LayoutChooser.styl

```diff
@@ -1,5 +1,34 @@
 $borderColor = rgba(77, 99, 110, 0.81)
 
+.pull-left
+  float: left!important;
```

Comment:
Are imported react styles like this scoped to the component/module, or do they bleed out? I'm unfamiliar here.

If they bleed out, this should be moved to a utility class file somewhere else.

Reply:
They can bleed out. I will change to be scoped only in this component